Makefile

```diff
@@ -216,7 +218,8 @@ build-aws-ubuntu16:
 
 build-aws-centos7: export IMAGE_NAME := $(IMAGE_NAME)
 
-build-aws-centos7: build-aws-centos7-base
+build-aws-centos7: 
+	@ METADATA_FILENAME_POSTFIX=$(METADATA_FILENAME_POSTFIX) make build-aws-centos7-base
```

Comment:
shouldn't we add this to other OS types also? I know we don't burn those right now, but if we will at some point we might forget about this.

Reply:
it will also work for other make targets, the extra variable needed only here because we are executing another make target only here and we need to ensure that this postfix will be the same when `make build-aws-centos7-base` are running